docs_user/modules/proc_configuring-networking-for-control-plane-services.adoc

```diff
@@ -57,53 +53,47 @@ spec:
       "ipam": {
         "type": "whereabouts",
         "range": "172.17.0.0/24",
-        "range_start": "172.17.0.20",
-        "range_end": "172.17.0.50",
-        "exclude": [
+        "range_start": "172.17.0.20", <1>
+        "range_end": "172.17.0.50", <2>
+        "exclude": [ <3>
           "172.17.0.24/32",
           "172.17.0.44/31"
         ]
       }
     }
-```
-
-The example above would exclude addresses `172.17.0.24` as well as
-`172.17.0.44` and `172.17.0.45` from the allocation pool.
-
-//== Load balancer IP addresses
-
-Some {OpenStackShort} services require load balancer IP addresses. These IP addresses
-belong to the same IP range as the control plane services, and are managed by
-MetalLB. The IP address pool is defined by `IPAllocationPool` CRs. This pool
-should also be aligned with the adopted configuration.
-
-For example,
-
-```
+----
++
+<1> Defines the start of the IP range.
+<2> Defines the end of the IP range.
+<3> Excludes part of the IP range. This example excludes IP addresses `172.17.0.24` as well as `172.17.0.44` and `172.17.0.45` from the allocation pool.
+
+. If your {OpenStackShort} services require load balancer IP addresses, define the pools for these services in an `IPAllocationPool` CR. For example: 
```

Comment:
The code example says “IPAddressPool” but the step says that you define a `IPAllocationPool` CR for load balancer IP addresses. Which one is correct?

Reply:
IPAddressPool: https://metallb.io/configuration/_advanced_ipaddresspool_configuration/